master/buildbot/scripts/dataspec.py

```diff
@@ -28,7 +28,8 @@
 @defer.inlineCallbacks
 def dataspec(config):
     master = yield fakemaster.make_master()
-    data = connector.DataConnector(master)
+    data = connector.DataConnector()
+    data.setServiceParent(master)
```

Comment:
@tardyp I added a line to fix the error occurs when performing `gulp dev` here as `DataConnector` no longer accept a master as init params.